### PR TITLE
Add a check for combining two paths with embedded EPREFIX

### PR DESCRIPTION
**Pull request by mgorny**
// NB: there's also a need for similar check for using `doins $(python_get_sitedir)` etc.

### CODE REVIEW

File: src/pkgcheck/checks/codingstyle.py [hunk 1]
```diff
@@ -203,6 +267,9 @@ def feed(self, entry):
             yield MissingSlash(pkg, var, lines)
         for var, lines in unnecessary.items():
             yield UnnecessarySlashStrip(pkg, var, lines)
+        print(double_prefix)
```
Comment (radhermit) [inline src/pkgcheck/checks/codingstyle.py]: I imagine this is a errant debugging line.

Comment (mgorny) [reply src/pkgcheck/checks/codingstyle.py]: Sorry, removed now.

File: src/pkgcheck/checks/codingstyle.py [hunk 2]
```diff
@@ -167,31 +167,95 @@ def short_desc(self):
...
 class PathVariablesCheck(base.Template):
     """Scan ebuild for path variables with various issues."""
 
     feed_type = base.ebuild_feed
-    known_results = (MissingSlash, UnnecessarySlashStrip)
-    variables = ('ROOT', 'EROOT', 'D', 'ED')
+    known_results = (MissingSlash, UnnecessarySlashStrip, DoublePrefixInPath)
+    prefixed_variables = ('EROOT', 'ED')
+    variables = ('ROOT', 'D') + prefixed_variables
+    prefixed_getters = (
```
Comment (radhermit) [inline src/pkgcheck/checks/codingstyle.py]: Would it be possible to flag these in a formatted way somehow using the eclass doc markup so we don't have to maintain the list manually in pkgcheck once eclass doc parsing support is written?

Comment (mgorny) [reply src/pkgcheck/checks/codingstyle.py]: Hmm, I don't see a problem with that. I suppose an extra tag for VARIABLE and FUNCTION thingies.

**Comment by radhermit** [comment]
Merged with a TODO note mentioning the above. Once eclass parsing is done, I'll return to poke you/whoever to fix that if it's not done by then. :smile:

**Pull request closed without merging**